code/modules/mob/living/living_defense.dm

```diff
@@ -134,6 +134,8 @@
 		return FALSE
 	if(!CHECK_BITFIELD(datum_flags, DF_ISPROCESSING))
 		return FALSE
+	if(get_fire_resist() <= 0 || get_hard_armor("fire", BODY_ZONE_CHEST) >= 100)	//having high fire resist makes you immune
```

Comment:
Did you change how resin jelly works? It's still magic-numbering fire_resist_modifier and removing the magic number check here breaks resin jelly.

Reply:
not clamping the resist value should do it since all instances of handling get_fire_resist catch multipliers <= 0

Reply:
It might be better to instead add Tivi's robotic trait TRAIT_NON_FLAMMABLE and remove once the jelly runs out.

I dont know if that will then nullify damage when moving across flames though.

Reply:
I'll see about it when I get around to test, if it doesn't work, maybe.

Reply:
Just crunched the math. -20 is plenty but eww, magic number.

Feel free to resolve this comment though. This will work.

Reply:
does in fact seem to work pretty well, in fact better than before I believe due to also affecting some other cases.